### PR TITLE
Make markers clickable and show popups

### PR DESCRIPTION
**Pull request by bkeese**
Fixes #2679

Makes markers clickable and adds popups.

Falls back to the MEJS2 style of marker that are below the time rail.
capture

### CODE REVIEW

File: app/assets/javascripts/media_player_wrapper/mejs4_helper_markers.es6 [hunk 1]
```diff
@@ -304,13 +289,13 @@ class MEJSMarkersHelper {
...
         // Insert the fresh HTML table
         $('#markers').replaceWith(response);
-        // Add event listeners to newly created row
-        t.addMarkersTableListeners();
         // hide marker sections if no markers (first row is header)
         if ($('#markers').find('tr').length === 1) {
           $('#markers_section').collapse('hide');
           $('#markers_heading').hide();
         } else {
+          // Add event listeners to newly created row
```
Comment (adamjarling) [inline app/assets/javascripts/media_player_wrapper/mejs4_helper_markers.es6]: This is an old comment hopefully, "to newly created row"?   It should be applying to all rows.

File: app/assets/javascripts/media_player_wrapper/mejs4_helper_markers.es6 [hunk 2]
```diff
@@ -358,27 +343,91 @@ class MEJSMarkersHelper {
...
    * Update markers in Mediaelement player's time rail by hooking into the Mediaelement Markers plugin
    * @function updateVisualMarkers
    * @param {Array} markers Array of marker start times
    * @return {void}
    */
-  updateVisualMarkers(markers) {
-    const t = this;
-    const player = mejs4AvalonPlayer.player;
-    player.options.markers = markers;
+  updateVisualMarkers() {
```
Comment (adamjarling) [inline app/assets/javascripts/media_player_wrapper/mejs4_helper_markers.es6]: If we're in a time crunch, it's fine but ideally we should break this function down some it seems.  Does it pass eslint?

File: app/assets/javascripts/media_player_wrapper/mejs4_plugin_playlist_items.es6 [hunk 3]
```diff
@@ -167,45 +167,46 @@ Object.assign(MediaElementPlayer.prototype, {
...
+        const url = `/media_objects/${el.dataset.mediaObjectId}/section/${id}`;
+
+        // Update mejs4AvalonPlayer.playlistItem with ids here
+        mejs4AvalonPlayer.playlistItem = Object.assign(
+          {},
+          mejs4AvalonPlayer.playlistItem,
+          { id: playlistItemId, playlist_id: playlistId, position: null }
+        );
+
+        // Update mejs4AvalonPlayer.playlistItem with ids here
```
Comment (adamjarling) [inline app/assets/javascripts/media_player_wrapper/mejs4_plugin_playlist_items.es6]: Is this code block a duplicate of the one above it?

**Comment by bkeese** [comment]
Something in the merge with develop broke things. Closing for now.

**Pull request closed without merging**